web/src/app/core/services/repository.service.ts

```diff
@@ -24,6 +25,7 @@ export class RepositoryService {
         private afs: AngularFirestore,
         private fns: AngularFireFunctions,
         private authService: AuthenticationService,
+        private spinnerService: SpinnerService
```

Comment:
Not important, but I think we end with a `,`?

Reply:
I will fix it